TwitchClient/Channel.swift

```diff
@@ -1,27 +0,0 @@
-//
-//  Channel.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/8/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-
-class Channel: NSObject {
-    
-    var status : String?
-    var language : String?
-    var displayName : String?
-    var logo : String?
-    
-    func populateChannelWithData(DataDictionary : NSDictionary) -> Channel
-    {
-        self.status = DataDictionary["status"] as? String
```

Comment:
Any strings or numbers should be in a constants file, preferably as an enum. So your class don't have *any* magic strings or numbers.

Topics: Magic string/number and clean code.